### PR TITLE
[metadata.movies.thetvdb.com.v4.python] v.1.0.10

### PR DESCRIPTION
**Pull request by romanvm**
### Description
The new TheTVDB movies scraper

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scrapers/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by KarellenX** [comment]
@romanvm 
You can probably merge these new PR's. There have been no issues reported since you posted this.

**Comment by KarellenX** [comment]
@razzeee 

Are you able to merge these TVDB PR's? Thanks.

File: metadata.movies.thetvdb.com.v4.python/addon.xml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    <description lang="vi_VN">TheTVDB.com là trình lấy dữ liệu TV show. Site chứa một số lượng khổng lồ cơ sở dữ liệu mở, có thể được thay đổi bởi mọi người và đầy đủ dữ liệu lớn các show trên nhiều ngôn ngữ. Tất cả nội dung và hình ảnh trên site này đều được đóng góp bởi toàn bộ người dùng của nó và có chất lượng cao. Mô hình cơ sở dữ liệu và website đều là mã nguồn mở được hỗ trợ bởi GPL.</description>
+    <description lang="zh_CN">TheTVDB.com 是一个电视刮削器。该网站是一个巨大的开放式数据库，任何人都可以修改，包含多语言的许多电视剧集资料。网站设定了高品质标准，所有图片和内容都由用户提供。数据库结构和网站均在 GPL 许可下开源。</description>
+    <description lang="zh_TW">TheTVDB.com是一個電視節目。該網站是一個龐大並可以由任何人修改的開放性資料庫，並包含可以以多國語言顯示的完整的數據資料。所有內容和圖片都是經由用戶提供使用，有一定的標準或質量。數據庫架構和網站是在GPL下的開放性源碼。</description>
+    <platform>all</platform>
+    <license>GPL-3.0-or-later</license>
+    <reuselanguageinvoker>false</reuselanguageinvoker>
+    <assets>
+      <icon>media/icon.png</icon>
+      <screenshot>media/screenshot-1.jpg</screenshot>
```
Comment (razzeee) [inline metadata.movies.thetvdb.com.v4.python/addon.xml:120]: Not sure if we usually allow screenshots of the website here

Comment (romanvm) [reply metadata.movies.thetvdb.com.v4.python/addon.xml:120]: On the other side, what screenshots are needed for a scraper? Should I suggest TheTVDB to remove all the screenshots?

Comment (razzeee) [reply metadata.movies.thetvdb.com.v4.python/addon.xml:120]: Yeah, I don't think any of the other scrapers have screenshots. Would like a 2nd opinion on this. @enen92

**Review by razzeee — APPROVED** [review]
Looks fine code wise, haven't runtime tested this yet.

File: metadata.movies.thetvdb.com.v4.python/addon.xml (hunk 1 above)
Comment (enen92) [reply metadata.movies.thetvdb.com.v4.python/addon.xml:120]: We allow, it's simply not mandatory

**Comment by KarellenX** [comment]
@razzeee 
Do you have the time to finalise the two TVDB scrapers?
Thanks.

**Comment by razzeee** [comment]
I think this is fine to go in, but I haven't really followed the discussions about the settings like age and gender.

**Comment by KarellenX** [comment]
Thanks @razzeee 

IIRC I don't think there was an issue. Do you remember @Darren-Hill or @keithah ?

**Comment by KarellenX** [comment]
@razzeee 

Darren doesn't know the answer. I don't know the answer either. 
What is the next step? Shall we let these scrapers sit here for another month or two while someone gets around to figuring it out?

**Comment by romanvm** [comment]
Currently demographic info is not mandatory and the scrapers work as they are.

**Comment by keithah** [comment]
I'd say let's merge them if they work fine!

**Pull request merged**